vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -912,11 +912,11 @@ void saveToJson(JsonWriter* writer, const WALLET_PROPERTIES_ST& data) {
   writer->String("altcurrency");
   writer->String(data.altcurrency_.c_str());
 
-  writer->String("probi_");
+  writer->String("probi");
   writer->String(data.probi_.c_str());
 
   writer->String("balance");
-  writer->Double(data.balance_);
+  writer->String(std::to_string(data.balance_).c_str());
```

Comment:
I agree using a String is the correct way, however right now the solution to be consistent is to change `d.HasMember("balance") && d["balance"].IsString()` to `d.HasMember("balance") && d["balance"].IsDouble()`

Reply:
I think we discussed on a call that we are going to change all double to string internally. I don't think we should change it there.

Reply:
structure currently have it as string so we should leave it as string for now